packages/packages/v1-adapters/src/listeners/utils.ts

```diff
@@ -0,0 +1,38 @@
+import { ListenerEvent } from './types';
+
+export function dispatchOnV1Ready() {
+	return getV1LoadingPromise().then( () => {
+		window.dispatchEvent( new CustomEvent( 'elementor/v1/initialized' ) );
+	} );
+}
+
+function getV1LoadingPromise() {
+	type ExtendedWindow = Window & {
```

Comment:
Consider using ts-ignore or just change to one-line casting.

Reply:
Done